tests/queries/0_stateless/02346_text_index_dictionary_cache.sql

```diff
@@ -61,17 +61,6 @@ SELECT count() FROM tab WHERE hasAnyTokens(message, 'text_127');
 SYSTEM FLUSH LOGS query_log;
 SELECT * FROM text_index_cache_stats(filter = 'text_127');
 
-SELECT '--- no profile events when cache is disabled.';
```

Comment:
Hmm, it would be nice to restore this test. When the cache is disabled, no cache hits/misses are expected.